src/applications/vaos/services/location/transformers.js

```diff
@@ -258,35 +259,10 @@ export function transformATLASLocation(tasInfo) {
  * @param {Array} params.directSupportedFacilities An array of location ids that support direct scheduling for a particular type of care
  * @returns {Array} A location resource
  */
-export function setSupportedSchedulingMethods({
-  location,
-  requestFacilities,
-  directFacilities,
-} = {}) {
+export function setSupportedSchedulingMethods({ location, settings } = {}) {
   const id = location.id;
-  const requestSupported = {};
-  const directSchedulingSupported = {};
-
-  const facilityRequestSettings = requestFacilities.find(
```

Comment:
Generally, this logic is now in the transformer, so that the data we add to the facility here is similar to the data we would get back to the VAOS service, and we don't have to transform that data as well.